### PR TITLE
Updated PNDA Guide for provisioning PNDA on OpenStack.

### PR DESCRIPTION
**Pull request by AbhilashS-MapleLabs**
PNDA-4641

### CODE REVIEW

File: provisioning/openstack/CONFIGURE.md [hunk 1]
```diff
@@ -16,54 +16,53 @@ Create or designate a suitable machine for running the PNDA CLI. We recommend Ce
 
 #### Obtain code
 
-Clone the [Heat templates repository](https://github.com/pndaproject/pnda-heat-templates) repository from the master branch at a specific release tag (e.g. ```release/3.5```) to the client machine.
+Clone the [pnda-cli repository](https://github.com/pndaproject/pnda-cli) repository from the master branch at a specific release tag (e.g. ```release/X.Y```, where X is major version > 4 and Y is a subversion) to the client machine.
```
Comment (trsmith2) [inline provisioning/openstack/CONFIGURE.md]: The additional text (where X is major, etc) here is misleading. There's no need to explain semantic versioning mid-sentence. The user simply needs to pick a valid release tag.

File: provisioning/openstack/CREATE.md [hunk 2]
```diff
@@ -14,49 +14,21 @@ To use the PNDA command line interface, you will need to install the python, hea
...
-./heat_cli.py
-usage: heat_cli.py [-h] [-y] [-e PNDA_CLUSTER] [-n DATANODES]
-                   [-o OPENTSDB_NODES] [-k KAFKA_NODES] [-z ZK_NODES]
-                   [-f {standard}] [-b BRANCH] [-s KEYPAIR]
-                   {create,destroy}
-```
-
-In particular note that the -s option refers to the name of the key pair created in the OpenStack tenant during the preparation phase.
+The pnda-cli.py script allows launching a PNDA deployment. It sits in the cli subdirectory.
+To launch PNDA on Openstack please proceed to [pndacli repository](https://github.com/pndaproject/pnda-cli).
```
Comment (trsmith2) [inline provisioning/openstack/CREATE.md]: Typo, should be pnda-cli

File: provisioning/openstack/CREATE.md [hunk 3]
```diff
@@ -14,49 +14,21 @@ To use the PNDA command line interface, you will need to install the python, hea
...
-
-```
-. <project>-openrc.sh
-```
 
 #### CLI invocation
 
-The heat_cli.py scripts allows to launch a PNDA deployment. It sits in the cli subdirectory.
-
-
```
Comment (trsmith2) [inline provisioning/openstack/CREATE.md]: You've completely removed an example invocation of the CLI, which is the primary purpose of this page, and deleted the statement about -s which is a major source of error.

Leave the existing text and make reference to the more detailed information on the repository. Experience shows that users need this information in its simple form all in one place, even if it means redundancy in the docs.

File: provisioning/openstack/PREPARE.md [hunk 4]
```diff
@@ -6,13 +6,13 @@
...
-- PNDA is deployed using Heat orchestration. We use `heat_template_version: 2014-10-16`. Please make sure this OpenStack service is available.
+- PNDA is regularly tested on OpenStack [Mitaka](http://releases.openstack.org) but is expected to work on versions of OpenStack from Kilo onwards.
+- PNDA is deployed using Heat orchestration. We use `heat_template_version: 2015-04-30`. Please make sure this OpenStack service is available.
 - PNDA makes use of the Swift object store. Please make sure this OpenStack service is available.
 - Before continuing, please ensure -
 	- At least one suitable tenant exists
 	- It provides sufficient resources and quota to provision one of the flavors described below
-	- Nova VM provisioning has been shown to work and you're able to access the VM and have the expected network connectivity (which will depend on what interfaces and networks you decide to configure).
+	- Openstack VM provisioning has been shown to work and you're able to access the VM and have the expected network connectivity (which will depend on what interfaces and networks you decide to configure).
```
Comment (trsmith2) [inline provisioning/openstack/PREPARE.md]: The term Nova was used here to mean that the user should try instantiating a VM manually, which is done through Nova. I don't think generalising this to 'OpenStack' really helps.

File: repos/README.md [hunk 5]
```diff
@@ -9,8 +9,7 @@ It consists of the following source code repositories and sub-projects:
 ### Provisioning
 
  * [platform-salt](http://github.com/pndaproject/platform-salt): provisioning logic for creating PNDA
- * [pnda-heat-templates](http://github.com/pndaproject/pnda-heat-templates): cluster templates for creating PNDA with Heat
- * [pnda-cli](http://github.com/pndaproject/pnda-cli): cluster templates for creating PNDA with CloudFormation on AWS
+ * [pnda-cli](http://github.com/pndaproject/pnda-cli): cluster templates for creating PNDA on AWS/OpenStack/Baremetal-Servers
```
Comment (trsmith2) [inline repos/README.md]: This would read better as 'orchestration application for creating PNDA on AWS, OpenStack or an existing pre-prepared cluster'

**Comment by AbhilashS-MapleLabs** [comment]
Will make the requested changes.

**Pull request merged**